src/js/components/HeaderMenu.js

```diff
@@ -45,13 +45,14 @@ export default class Menu extends React.PureComponent {
               Index
             </Link>
           </li>
-          {/* // TODO Enable when desktop feature is ready
-          <li>
-            <Link to="/sync" onClick={toggleMenu}>
-              Sync
-            </Link>
-          </li>
-          */}
+          {
+            /* // TODO Enable when desktop feature is ready */
```

Comment:
We can remove the todo

Reply:
I was actually waiting on the desktop feature to complete before removing the comment, exposed it so that sttm-desktop team can test while working on it.